awslabs/kubernetes-iteration-toolkit
# Testbed kit addon installation; sync tests;

*Issue #, if available:*

*Description of changes:*
- testworklows sync using flux;
- add tekton triggers rbac;
- kit addon installation using flux;
- refacor;
- bug-fixes;
- update eksctl image.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
